lib/_stream_readable.js

```diff
@@ -103,6 +103,7 @@ function ReadableState(options, stream, isDuplex) {
   this.ended = false;
   this.endEmitted = false;
   this.reading = false;
+  this.bytesRead = this.objectMode ? null : 0;
```

Comment:
This initialize the field for `readableState`, but later in the code we do `stream.bytesRead`. I fear this is going to have bugs.